<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSettings</class>
 <widget class="QWidget" name="WidgetSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBasicSettings">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="layBasic">
       <item>
        <widget class="QLabel" name="lblNick">
         <property name="text">
          <string>Nick</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtNick"/>
       </item>
       <item>
        <widget class="QLabel" name="lblIncoming">
         <property name="text">
          <string>Incoming folders</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layIncoming">
         <item>
          <widget class="QListView" name="lstIncoming"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="layIncomingButtons">
           <item>
            <widget class="QPushButton" name="butAddIncoming">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/icons/ressources/add.png</normaloff>:/icons/ressources/add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="butRemoveIncoming">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/icons/ressources/delete.png</normaloff>:/icons/ressources/delete.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacerIncoming">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lblShared">
         <property name="text">
          <string>Shared folders</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layShared">
         <item>
          <widget class="QListView" name="lstShared"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="laySharedButtons" stretch="0,0,0">
           <item>
            <widget class="QPushButton" name="butAddShared">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/icons/ressources/add.png</normaloff>:/icons/ressources/add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="butRemoveShared">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/icons/ressources/delete.png</normaloff>:/icons/ressources/delete.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacerShared">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvancedSettings">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="layAdvanced">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="lblCoreAddress">
         <property name="text">
          <string>Core address</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layCoreAddress">
         <item>
          <widget class="QLineEdit" name="txtCoreAddress"/>
         </item>
         <item>
          <widget class="QPushButton" name="butResetCoreAddress">
           <property name="text">
            <string>Reset to local</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lblPassword">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPassword"/>
       </item>
       <item>
        <spacer name="spacerAdvancedSettings">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
